### PR TITLE
Added transform guide

### PR DESCRIPTION
**Pull request by Abdulmumin1**
Added list of shortcuts to transform an object in blender!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,15 @@ This list all Windows shortcuts for Blender 3.0+
...
 | <kbd>ALT</kbd>+<kbd>SPACE</kbd>  | Opens a menu with manipulator orientation choices                                        |
 
+## Transform
+
+|               Keys               | Action                                                                                   |
+| :------------------------------: | :--------------------------------------------------------------------------------------- |
+|          <kbd>G</kbd>          | Grab object                                             |
+|         <kbd>S</kbd>         | scale object                                                               |
+| <kbd>R</kbd> | Rotate object |
+| <kbd>x</kbd>, <kbd>y</kbd> or <kbd>z</kbd> | to `rotate` or `scale` in the relative direction |
```
Comment (Panquesito7) [inline README.md]:
```suggestion
| <kbd>x</kbd>, <kbd>y</kbd> or <kbd>z</kbd> | To `rotate` or `scale` in the relative direction |
```

File: README.md [hunk 2]
```diff
@@ -40,6 +40,15 @@ This list all Windows shortcuts for Blender 3.0+
 | <kbd>CTRL</kbd>+<kbd>SPACE</kbd> | Opens menu with manipulator type choices (Press 2, 3, 4, or 5 to change the widget type) |
 | <kbd>ALT</kbd>+<kbd>SPACE</kbd>  | Opens a menu with manipulator orientation choices                                        |
 
+## Transform
+
+|               Keys               | Action                                                                                   |
+| :------------------------------: | :--------------------------------------------------------------------------------------- |
+|          <kbd>G</kbd>          | Grab object                                             |
+|         <kbd>S</kbd>         | scale object                                                               |
```
Comment (Panquesito7) [inline README.md]:
```suggestion
|         <kbd>S</kbd>         | Scale the selected object                                                               |
```

File: README.md [hunk 3]
```diff
@@ -40,6 +40,15 @@ This list all Windows shortcuts for Blender 3.0+
 | <kbd>CTRL</kbd>+<kbd>SPACE</kbd> | Opens menu with manipulator type choices (Press 2, 3, 4, or 5 to change the widget type) |
 | <kbd>ALT</kbd>+<kbd>SPACE</kbd>  | Opens a menu with manipulator orientation choices                                        |
 
+## Transform
```
Comment (Panquesito7) [inline README.md]: What do you think of this emoji?
```suggestion
## 🧊 Transform
```

**Comment by Abdulmumin1** [comment]
Thanks for the correction, i was focusing on hactoberfest instead of the overall value of the project.
my bad 😅.

**Review by mkubdev — APPROVED** [review]
Hello 👋🏻 ! Thanks for your contribution, LGTM ⚡

**Pull request merged**